Cargo.toml

```diff
@@ -62,4 +62,4 @@ required-features = ["proj"]
 [dev-dependencies]
 approx = "0.3.2"
 rust_decimal_macros = "1.0.1"
-transit_model_builder= "0.1.0"
+transit_model_builder = { path = "./model-builder" }
```

Comment:
How about doing the same for the dependency `transit_model_procmacro` from
```
transit_model_procmacro = { version = "0.1.0", path = "./transit_model_procmacro" }
```
to
```
transit_model_procmacro = { path = "./transit_model_procmacro" }
```

Reply:
After discussion, only the `dev-dependencies` have been changed, the other, since we're not sure yet of the impact during release, we do not touch for now.